File: api/code-samples/javascript/klaytnDebug/blockchainInspection/printBlock.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const blockNumber = 65120

    sdk.debug.printBlock(blockNumber, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/blockchainInspection/printBlock.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockNumber = "0x80"

sdk = OpenSDK(host)
debug_response = sdk.debug.print_block(blockNumber)

print(json.loads(debug_response.response.data))
